commons/src/types/ScheduleEditor.ts

```diff
@@ -62,6 +63,7 @@ interface HostRules {
 interface SectionOptions {
   expanded: boolean;
   editable: boolean;
+  hidden_fields?: string[];
```

Comment:
Where is this field being used in the code? Can't find them in the changes

Reply:
Note the `TODO` in the readme for this PR; that's going to be addressed in #410